build.sbt

```diff
@@ -185,6 +185,20 @@ lazy val twirl = http4sProject("twirl")
   .enablePlugins(SbtTwirl)
   .dependsOn(core % "compile;test->test")
 
+lazy val zipkin = http4sProject("zipkin")
+  .settings(
+    description := "Zipkin support for http4s"
+  )
+  .dependsOn(
+    core % "compile->compile;test->test",
+    theDsl % "compile->compile;test->test",
+    server % "compile->compile;test->test",
+    client % "compile->compile;test->test",
+    blazeServer % "compile->compile;test->test",
+    blazeClient % "compile->compile;test->test",
+    argonaut % "compile->compile;test->test"
```

Comment:
As the master build moves toward cats (#390), we're standardizing on circe for places we need a concrete JSON library (#584) .  I suppose argonaut makes sense for now, while master is on scalaz-stream, but this will need to be refactored in the medium term.